packages/noya-state/src/reducers/applicationReducer.ts

```diff
@@ -51,6 +53,13 @@ export type SelectedControlPoint = {
   controlPointType: controlPointType;
 };
 
+export type SelectedGradient = {
+  layerId: string;
+  fillIndex: number;
+  stopIndex: number;
+  styleType: 'fill' | 'border';
```

Comment:
I just realized, if we change this to `'fills' | 'borders'` we, can simplify some of our other code by using the `styleType` directly to look up the correct property, e.g.:

```ts
if (layer.style?.[styleType]?.[fillIndex].fillType !== Sketch.FillType.Gradient) { return state }

const gradientStops = layer.style?.[styleType]?.[fillIndex].gradient.stops;
```